semcore/notice/src/Notice.jsx

```diff
@@ -36,6 +36,16 @@ class RootNotice extends Component {
     locale: 'en',
   };
 
+  constructor(props) {
+    super(props);
+
+    logger.warn(
+      !props['aria-label'],
+      'Provide aria-label to help identify the type of notification',
```

Comment:
```suggestion
      'Provide unique aria-label to help identify the type of notification',
```
probably better to clarify